test/python/WMCore_t/Services_t/Dashboard_t/DashboardAPI_t.py

```diff
@@ -32,12 +31,9 @@ def testApmonInstance(self):
 
         Just test initialization of apmon Instance
         """
-        dashboard = DashboardAPI()
-        apmonInst = dashboard.getApMonInstance()
-        self.assertTrue(apmonInst.initializedOK())
-        # Free up apmon instance and check if it was successfull
-        apmonInst.free()
-        self.assertEqual(None, apmonInst)
+        with DashboardAPI() as dashboard:
+            self.assertTrue(dashboard.apmon.initializedOK())
+        self.assertTrue(dashboard.apmon.freedOK())
```

Comment:
Does this make sense? I would guess "dashboard" is only valid within the context. But maybe you can do this.